roles/swift-account/tasks/main.yml

```diff
@@ -11,7 +11,6 @@
 - name: account-server config
   template: src=etc/swift/account-server.conf mode=0640
             dest=/etc/swift/account-server.conf owner=swift group=swift
-  notify: restart swift-account-services
 
```

Comment:
I'm not sure on this. There are three "upgrade" paths.

1) new major openstack version, where dbs and synced and such. `upgrade.yml` playbook.

2) Minor package version changes, where we want to stop all old code and make sure all new code is started. `site.yml` with `-e upgrade=true`

3) config file changes with the same package version. `site.yml`. So in the case of pushing a deployment out where account-server.conf contents change, but we're not changing packages, we still want things to restart.